src/super_gradients/examples/quantization/resnet_quantization_examples.py

```diff
@@ -0,0 +1,103 @@
+from collections import OrderedDict
```

Comment:
Remove/remake the entire file please. Many irrelevant old parts which would crash.